<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTMLFormatterWidget</class>
 <widget class="QWidget" name="HTMLFormatterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Indentations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="formatButton">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="toolTip">
        <string>Open</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditCopy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditPaste"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditClear"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="placeholderText">
      <string>Put HTML code here</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
